src/hooks/tradingView/useTradingView.ts

```diff
@@ -146,6 +146,8 @@ export const useTradingView = ({
     if (marketId && tickSizeDecimals !== undefined && !tvWidget) {
       const widgetOptions = getWidgetOptions();
       const widgetOverrides = getWidgetOverrides({ appTheme, appColorMode });
+      const languageCode =
+        SUPPORTED_LOCALES.find(({ locale }) => locale === selectedLocale)?.baseTag ?? 'en';
```

Comment:
I used to entertain myself by trying to make maximally functional and type safe version of this kind of statement. Like imagine a `SUPPORTED_LOCALES.find(property('locale').eq(selecedLocale))?.baseTag`